Sources/PubNubChatComponents/Message List/MessageListViewController.swift

```diff
@@ -36,15 +36,39 @@ import InputBarAccessoryView
 import PubNub
 import PubNubChat
 
-open class ChatViewController: CollectionViewComponent {
+open class ChatViewController<ModelData, ManagedEntities>: CollectionViewComponent, UIViewControllerTransitioningDelegate
+  where ModelData: ChatCustomData,
+    ManagedEntities: ChatViewModels,
+    ManagedEntities: ManagedChatEntities,
+    ManagedEntities.Channel.MemberViewModel == ManagedEntities.Member,
+    ManagedEntities.Message.MessageActionModel == ManagedEntities.MessageAction
+{
   
+  public var cancellables = Set<AnyCancellable>()
+
   // Message Input Component
 
   public let messageInputComponent: MessageInputComponent
   private var messageInputComponentBottomConstraint: NSLayoutConstraint?
   
+  // Message List Component View Model
+  
+  private var messageListViewModel: MessageListComponentViewModel<ModelData, ManagedEntities>? {
+    get {
+      return viewModel as? MessageListComponentViewModel<ModelData, ManagedEntities>
```

Comment:
I think return here is not necessary

Reply:
That's true, I can change it. However, I think it would be great to change it everywhere if we agreed to skip `return` from any single-expression function. I can create a JIRA issue if we all agree on this new approach.